pennylane/tape/tape.py

```diff
@@ -219,6 +219,7 @@ class QuantumTape(AnnotatedQueue):
     """A quantum tape recorder, that records, validates and executes variational quantum programs.
 
     Args:
+        queue (Iterable): iterable of tape contents
```

Comment:
I wonder if there is a better word for this, since 'queue' is quite a loaded term already in PL?

Reply:
What you pass in there becomes the `queue` for the tape, so I think using a different word would be confusing and misleading.